Rendering/export broken with version > 3.0

Hi,
I'm coming across some errors making the extension not usable since the version 3.0.0 (2.03 being the last working version )

this is a screen shot of what I have in the side panel or in the browser
![image](https://user-images.githubusercontent.com/1909018/52501113-958ea180-2bad-11e9-9386-b52a5d181445.png)


and the following errors inside the chrome console: 
![image](https://user-images.githubusercontent.com/1909018/52500673-95da6d00-2bac-11e9-92e9-2750f0f12004.png)


My setup: 
Version: 1.31.0 (user setup)
Commit: 7c66f58312b48ed8ca4e387ebd9ffe9605332caa
Date: 2019-02-05T22:35:56.624Z
Electron: 3.1.2
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134

@evilz I've tried to install the new version but I can not generate the vsix file to install it locally. I get the error:

```
Executing prepublish script 'npm run vscode:prepublish'...
 ERROR  Command failed: npm run vscode:prepublish
sh: 1: tsc: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! vscode-reveal@3.3.0 vscode:prepublish: `tsc -p ./`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the vscode-reveal@3.3.0 vscode:prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```
I work with Linux (Kubuntu) and I have installed node version 10.15.1 and npm version 6.4.1.

The solution proposed by @distantcam does not work in my case.

--- a/src/Configuration.ts
+++ b/src/Configuration.ts
@@ -91,8 +91,8 @@ export const defaultConfiguration: Configuration = {
   description: '',
   author: '',
   notesSeparator: 'note:',
-  separator: '^\r?\n---\r?\n$',
-  verticalSeparator: '^\r?\n--\r?\n$',
+  separator: '^\\r?\\n---\\r?\\n$',
+  verticalSeparator: '^\\r?\\n--\\r?\\n$',
 
   customHighlightTheme: null,
   customTheme: null,

